src/main/java/software/amazon/msk/auth/iam/IAMOAuthBearerToken.java

```diff
@@ -61,15 +59,12 @@ public IAMOAuthBearerToken(String token) throws URISyntaxException {
         byte[] decodedBytes = Base64.getUrlDecoder().decode(tokenBytes);
         final String decodedPresignedUrl = new String(decodedBytes, StandardCharsets.UTF_8);
         final URI uri = new URI(decodedPresignedUrl);
-        List<NameValuePair> params = URLEncodedUtils.parse(uri, StandardCharsets.UTF_8);
```

Comment:
Why can't we use this anymore?

Reply:
Hi @sidyag , by removing AWS SDK V1, some transitive dependencies have been removed, then `NameValuePair` and `URLEncodedUtils` are no more available. I can have access again by adding `org.apache.httpcomponents:httpclient:4.5.+` dependency if needed. I re-implemented the method to avoid adding the whole dependency